tasks/record-cfd-version-in-manifest/task

```diff
@@ -5,11 +5,15 @@ set -exu
 git clone cf-deployment-release-candidate cf-deployment-rc-with-updated-version
 
 new_version="v$(cat cf-deployment-version/version)"
+new_version_major="$(cut -d . -f 1 cf-deployment-version/version)"
 
 pushd cf-deployment-rc-with-updated-version
   old_version="$(bosh int cf-deployment.yml --path /manifest_version)"
+  old_version_major="$(echo "${old_version}" | cut -d . -f 1 | sed 's/v//')"
 
   sed -i "s/manifest_version: ${old_version}/manifest_version: ${new_version}/g" cf-deployment.yml
+  sed -i "s/build: ${old_version}/build: ${new_version}/g" cf-deployment.yml
+  sed -i "s/version: ${old_version_major}/version: ${new_version_major}/g" cf-deployment.yml
```

Comment:
@jochenehret suggested that these new `sed` commands may be too generic for [cf-deployment.yml](https://github.com/cloudfoundry/cf-deployment/blob/main/cf-deployment.yml), and could unwittingly update lines that we don't want to update here.

Re-reading them I tend to agree. The `version` find-and-replace in particular could very easily accidentally catch an entry in `releases`.

I should add a comment (i.e. "# auto-generated") to the lines added in https://github.com/cloudfoundry/cf-deployment/pull/1101 (and possibly the `manifest_version` field as well) to ensure we always select and update the correct lines.